docs/source/topics/loggers.rst

```diff
@@ -0,0 +1,34 @@
+=======
+Logging
+=======
+
+Frontera is using Python native logging system. That means you can manage what messages are logged and where by writing
```

Comment:
I would rephrase this as :
Frontera uses Python native logging system. This allows a user to manage logged messages by writing a logger configuration file (see :setting:`LOGGING_CONFIG`).